homeassistant/components/sensirion_ble/sensor.py

```diff
@@ -0,0 +1,131 @@
+"""Support for Sensirion sensors."""
+from __future__ import annotations
+
+from typing import Optional, Union
+
+from sensor_state_data import (
+    DeviceKey,
+    SensorDescription,
+    SensorDeviceClass as SSDSensorDeviceClass,
+    SensorUpdate,
+    Units,
+)
+
+from homeassistant import config_entries, const
+from homeassistant.components.bluetooth.passive_update_processor import (
+    PassiveBluetoothDataProcessor,
+    PassiveBluetoothDataUpdate,
+    PassiveBluetoothEntityKey,
+    PassiveBluetoothProcessorCoordinator,
+    PassiveBluetoothProcessorEntity,
+)
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.sensor import sensor_device_info_to_hass_device_info
+
+from .const import DOMAIN
+
+SENSOR_DESCRIPTIONS: dict[
+    tuple[SSDSensorDeviceClass, Units | None], SensorEntityDescription
+] = {
+    (
+        SSDSensorDeviceClass.CO2,
+        Units.CONCENTRATION_PARTS_PER_MILLION,
+    ): SensorEntityDescription(
+        key=f"{SensorDeviceClass.CO2}_{Units.CONCENTRATION_PARTS_PER_MILLION}",
```

Comment:
Just realized the key is using the Home Assistant class and it should be 
`SSDSensorDeviceClass.CO2` instead. Same for the rest.

Sorry I missed it on the first pass. Please address in a fresh PR.

Thanks 👍

Reply:
That doesn't actually matter much since the value is the same when stringified, but sure :)